### ISSUE TITLE
[Missing label in react]: No way to pass label to MultiSelect.Filterable input

### ISSUE THREAD

**Issue by jamesjmcconnell** [carbon-design-system/carbon#5397]
## Detailed description

> What version of the Carbon Design System are you using?
carbon-components-react v6.x

I cant seem to find a way to pass a label to the MultiSelect.Filterable component in carbon components react v6.x, the lighthouse report for my page expects there to be a label for every input and the input in the component doesnt have an associated label (https://github.com/carbon-design-system/carbon-components-react/blob/v6/src/components/MultiSelect/FilterableMultiSelect.js#L291)

whereas the v7.x version of the component does https://github.com/carbon-design-system/carbon-components-react/blob/master/src/components/MultiSelect/FilterableMultiSelect.js#L281

**Comment by abbeyhrt** [comment]
Hi @jamesjmcconnell! 👋 Thank you for making this issue! We aren't currently resolving any accessibility issues for v6 of carbon-components-react but you are welcome to contribute a PR and we will work quickly on getting it merged in! If you have any other questions, let me know!
